FrameworkInternals/transformDesign.py

```diff
@@ -185,7 +185,10 @@ def transformDesign(xsltTransformation, outputFile, requiresMerge, astyleRun, ad
 
         if astyleRun:
             try:
-                subprocess.call([getCommand('astyle'), outputFile])
+                return_code = subprocess.call([getCommand('astyle'), outputFile])
```

Comment:
any profit in having the return_code in the logged error message ?

Reply:
I don't think so, astyle do not seem to have specific return codes.
But I can add it to the message, no big deal.
I understand there are no more remarks on this one.